src/b-splines/b-splines.jl

```diff
@@ -73,9 +74,16 @@ function interpolate!{IT<:DimSpec{BSpline},GT<:DimSpec{GridType}}(A::AbstractArr
     interpolate!(tweight(A), A, it, gt)
 end
 
+immutable Cubic{BC<:Flag} <: Degree{3} end
+Cubic{BC<:Flag}(::BC) = Cubic{BC}()
+
+immutable Quadratic{BC<:Flag} <: Degree{2} end
+Quadratic{BC<:Flag}(::BC) = Quadratic{BC}()
```

Comment:
I'm not sure why these moved here, rather than staying in `quadratic.jl` and `cubic.jl`. I have no strong opinion on the matter, but *if* they move, `Linear` and `Constant` should move here too.